pkg/unittest/test_suite.go

```diff
@@ -328,35 +332,44 @@ func (s *TestSuite) polishChartSettings(test *TestJob) {
 	log.WithField(common.LOG_TEST_SUITE, "polish-chart-settings").Debug("test.chart '", test.Chart)
 }
 
+type SuiteResult struct {
+	Pass       bool
+	FailFast   bool
+	JobResults []*results.TestJobResult
+}
+
 func (s *TestSuite) runV3TestJobs(
 	chartPath string,
 	cache *snapshot.Cache,
-	failfast bool,
+	failFast bool,
 	renderPath string,
-) (bool, []*results.TestJobResult) {
-	suitePass := false
+) *SuiteResult {
+	result := SuiteResult{Pass: true, FailFast: false}
```

Comment:
I would prefer to have the default value of Pass to be false, so that the testresults are failing when something unexpected happens.

Reply:
Make sense. Changed